From: Daniel_Burrows@alumni.brown.edu <Daniel_Burrows@alumni.brown.edu>
Date: Sat, 19 Jun 2010 11:22:05 -0700
Subject: [PATCH] Don't try to test the search input controller if the GTK+ frontend wasn't built.
 If the GTK+ frontend isn't built, neither is the controller,
 so the test won't build.

---
 tests/test_search_input_controller.cc |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/tests/test_search_input_controller.cc b/tests/test_search_input_controller.cc
index 8b01da0..7704828 100644
--- a/tests/test_search_input_controller.cc
+++ b/tests/test_search_input_controller.cc
@@ -17,6 +17,8 @@
 // the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 // Boston, MA 02111-1307, USA.
 
+#ifdef HAVE_GTK
+
 #include <generic/apt/matching/compare_patterns.h>
 #include <generic/apt/matching/pattern.h>
 #include <gtk/controllers/search_input.h>
@@ -146,3 +148,5 @@ BOOST_FIXTURE_TEST_CASE(testEnteringCorrectTextSearches, SearchInputTest)
 
   get_controller()->enter_text(p_text);
 }
+
+#endif // HAVE_GTK
-- 
